src/main/java/io/swagger/oas/inflector/controllers/OpenAPIOperationController.java

```diff
@@ -289,15 +297,186 @@ public Response apply(ContainerRequestContext ctx) {
+                                                    }
+                                                }
+                                            }
+                                        }
+                                        String filename = extractFilenameFromHeaders(headers);
+                                        if (filename != null) {
+                                            try {
+                                                File file = new File(Files.createTempDir(), filename);
+                                                file.deleteOnExit();
+                                                file.getParentFile().deleteOnExit();
+                                                FileOutputStream fo = new FileOutputStream(file);
+                                                multipartStream.readBodyData(fo);
+                                                inputStreams.put(name, file);
+                                            } catch (Exception e) {
+                                                LOGGER.error("Failed to extract uploaded file", e);
+                                            }
+                                        } else {
+                                            ByteArrayOutputStream bo = new ByteArrayOutputStream();
+                                            multipartStream.readBodyData(bo);
+                                            String value = bo.toString();
+                                            headers.put(name, value);
+                                        }
+                                        if (name != null) {
+                                            formMap.put(name, headers);
+                                        }
+                                        headers = new HashMap<>();
+                                        name = null;
+                                        nextPart = multipartStream.readBoundary();
+                                    }
+                                } catch (IOException e) {
+                                    e.printStackTrace();
+                                }
+                            }
+                        } else {
+                            try {
+                                formDataString = IOUtils.toString(ctx.getEntityStream(), "UTF-8");
+                                parts = formDataString.split("&");
+
+                                for (String part : parts) {
+                                    String[] kv = part.split("=");
+                                    existingKeys.add(kv[0] + ": fp");
+                                }
+                            } catch (IOException e) {
+                                e.printStackTrace();
+                            }
+                        }
+                        //TODO HERE
+                        try {
+                            io.swagger.v3.oas.models.media.MediaType media = body.getContent().get(mediaType);
```

Comment:
what happen with "media" object if the mediaType string has a not valid value?

Reply:
done